src/bilder/images/dagster/templates/docker-compose.yaml.j2

```diff
@@ -46,13 +48,38 @@ services:
     - "{{ context.edx_pipeline_definition_directory }}:/etc/dagster/:ro"
     env_file:
       - .env
-  nginx:
-    image: nginx
+  traefik:
+    image: traefik:latest
+    container_name: traefik
     restart: unless-stopped
+    command:
+      - "--api.insecure=true"
+      - "--providers.docker"
+      - "--providers.docker.exposedbydefault=false"
+      - "--entryPoints.http.address=:80"
+      - "--entryPoints.http.http.redirections.entryPoint.to=https"
+      - "--entryPoints.http.http.redirections.entryPoint.scheme=https"
+      - "--entryPoints.https.address=:443"
+      - "--log.level=WARN"
+      - "--accesslog=true"
+      - "--providers.file.filename=/etc/traefik/traefik.yml"
     ports:
-    - {{ context.listener_port }}:{{ context.listener_port }}
+      - "80:80"
+      - "443:443"
+      - "8080:8080"
     volumes:
-    - {{ context.certificate_file }}:/etc/nginx/nginx.pem:ro
-    - {{ context.certificate_key_file }}:/etc/nginx/nginx-key.pem:ro
-    - {{ context.nginx_directory }}/nginx.conf:/etc/nginx/conf.d/dagster.conf:ro
-    - {{ context.nginx_directory }}/htpasswd:/etc/nginx/htpasswd:ro
+      - "{{ context.traefik_directory}}/traefik.yaml:/etc/traefik/traefik.yaml:ro"
+      - {{ context.certificate_file }}:/etc/traefik/odl-wildcard.pem:ro
+      - {{ context.certificate_key_file }}:/etc/traefik/odl-wildcard-key.pem:ro
```

Comment:
Maybe wrap these two lines in double quotes just so they are the same as 71 and 74?